src/components/Schedule/Schedule.jsx

```diff
@@ -85,37 +85,21 @@ function Schedule() {
 								</div>
 							</div>
 						</div>
-						<Stack sx={{ width: "100%" }}>
+						<div className="event-details">
 							<div className="event-title">
 								{t("schedule.event3.title")}
-								<Button
+								<button
+									className="schedule-btn"
 									disabled
-									sx={{
-										background: "var(--button-gradient)",
-										borderRadius: "100px",
-										color: "var(--medium-primary-color)",
-										mt: matchesSM ? "7vw" : matchesMD ? "3vw" : "0vw",
-										mb: matchesSM ? "7vw" : matchesMD ? "3vw" : "0vw",
-										p: "7px",
-										textTransform: "none",
-										minWidth: "16%",
-										width: matchesSM ? "50%" : matchesLG ? "25%" : undefined,
-										transition: "1s",
-										boxShadow:
-											"rgba(0, 0, 0, 0.15) 0px 15px 25px, rgba(0, 0, 0, 0.05) 0px 5px 10px",
-										"&:hover": {
-											backgroundColor: "#f094bc",
-										},
-									}}
 								>
 									<p className="button-text">{t("schedule.event3.status")}</p>
-								</Button>
+								</button>
```

Comment:
Is there a reason not to use the button component?

Reply:
Yeah, that's good. I was referring to using https://github.com/HacktheHill/2023.hackthehill.com/blob/main/src/components/Button/Button.jsx instead of an HTML button.